polls/tests.py

```diff
@@ -1,2 +1,54 @@
+import datetime
+
 from django.test import TestCase
+from django.utils import timezone
+from django.urls import reverse
+
+from .models import Question, Choice
+
+def create_question(question_text, days):
```

Comment:
```suggestion

def create_question(question_text, days):
```
Two line breaks to separate outer class/method.

Reply:
Made the change